pages/content/amp-dev/documentation/guides-and-tutorials/contribute/contribute-documentation/documentation-types.md

```diff
@@ -0,0 +1,274 @@
+   <td>Describe in exhaustive detail. 
+   </td>
+  </tr>
+  <tr>
+   <td>Provide code samples and real-world examples.
+   </td>
+   <td>Create an entire app.
+   </td>
+  </tr>
+  <tr>
+   <td>List technical uses and restraints.
+   </td>
+   <td>List every possible technical use and how it’s done. 
+   </td>
+  </tr>
+</table>
+
+
+
+## Concept Guide
+
+Concept guides help developers build a deeper understanding of AMP. A concept guide is like a topographic map. It shows the various trails in the area with details such as elevation changes, but it doesn’t prescribe a specific route through the terrain. Explain what a feature is and how it functions rather than how to build a feature.
+
+amp.dev examples:
+
+
+
+*   [Animate & transition]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/develop/animations/triggering_css_animations.md', locale=doc.locale).url.path}}?format=websites)
+*   [Track engagement with analytics]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/optimize-and-measure/configure-analytics.md', locale=doc.locale).url.path}}?format=websites)
+*   [Style & layout]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/develop/style_and_layout.md', locale=doc.locale).url.path}}?format=websites)
+
+<table>
+  <tr>
+   <td>
+<strong>Do</strong>
+   </td>
+   <td><strong>Don’t</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Provide the developer with all the elements needed to construct a solution.
+   </td>
+   <td>Actively guide the developer to specific end state.
+   </td>
+  </tr>
+  <tr>
+   <td>Cover all aspects of the subject areas.
+   </td>
+   <td>Focus on a specific task. 
+   </td>
+  </tr>
+  <tr>
+   <td>Include visual aids, such as diagrams or screenshots.
+   </td>
+   <td>Overthink this, you can request help for visual aids.
+   </td>
+  </tr>
+  <tr>
+   <td>Provide code samples and link out to other guides. 
+   </td>
+   <td>Provide a download to a finished project or stray off topic. 
+   </td>
+  </tr>
+</table>
+
+
+
+## Reference Documentation
+
+Reference documentation lists all the programmatic elements for an AMP component. It provides detailed behavioral information and designed for scanning. Reference documentation should include exemplary code samples and demonstrate use-cases.   
+
+amp.dev examples are found under [the AMP component catalogue]({{g.doc('/content/amp-dev/documentation/components.md', locale=doc.locale).url.path}}?format=websites).
+
+<table>
+  <tr>
+   <td><strong>Do</strong>
+   </td>
+   <td><strong>Don’t</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Use clear and concise language that explains how the component works.
+   </td>
+   <td>Explain a process or build a project.
+   </td>
+  </tr>
+  <tr>
+   <td>Structure with easy to scan titles, headings, and subheadings. 
+   </td>
+   <td>Group content under abstract names.
+   </td>
+  </tr>
+  <tr>
+   <td>Use present tense and active voice. 
+   </td>
+   <td>Avoid passive voice and future tense. 
```

Comment:
That's true for all types of docs, no? Or is this more relevant for reference docs?

Reply:
This has been flagged a few times. Deff true for all docs - but reference docs are extra guilty. A lot of writers just migrate the design doc into reference docs and its all "will do x" instead of "this does x". But I may remove it from this section since everyone is getting confused...

Reply:
ah got it! With an extra clarification it works, I'll leave it up to you :)